extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -155,6 +176,14 @@ export class AmpGeo extends AMP.BaseElement {
     return shouldBeTrueish;
   }
 
+  /**
+   * @return {string}
+   * @private
+   */
+  getHotPatchCountry_() {
```

Comment:
is this just so we can mock the results?

Reply:
yes actually the main reason for that is for  HotPatch mode tests